packages/malloy-vscode/src/server/completions/completion_docs.ts

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ */
+
+const DOCS_ROOT = "https://looker-open-source.github.io/malloy/documentation";
+
+export const COMPLETION_DOCS: {
```

Comment:
This is by no means intended to be the final form of this mini docs segment. Among other things, it would be nice if these snippets were testable, and if the links were guaranteed to exist in the documentation. However, it might be okay to not overthink this too much for the time being. 

@anikaks Would you mind looking through this and seeing if there's anything you would change here?